### PR TITLE
Disclaimer

### PR DESCRIPTION
**Pull request by danrot**
This PR adds a disclaimer page, which we absolutely need before we go online (credits to the photographer and so on).

When developing the next pages we should probably extract some of the code from the disclaimer page.

### CODE REVIEW

File: pages/disclaimer.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    render() {
+        return (
+            <Page>
+                <HeaderImage />
+                <Main>
+                    <HeaderSection>
+                        <h2>Disclaimer with some aditional stuff</h2>
+                    </HeaderSection>
+                    <MainSection>
+                        <h3>Information</h3>
```
Comment (danrot) [inline pages/disclaimer.js]: We need to change that to the real text.

File: pages/disclaimer.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+import styled from 'styled-components';
+import Page from '../components/Page.js';
+
+const HeaderImage = styled.header`
+    position: relative;
+    z-index: 1;
+    margin: 60px 60px 0 60px;
+    height: calc(100% - 115px);
+
+    background: url(http://placehold.it/1300x660) no-repeat center;
```
Comment (danrot) [inline pages/disclaimer.js]: Use the correct background image instead.

**Comment by danrot** [comment]
I think after merging this PR we have something we could potentially put online.

**Comment by tomsvogel** [comment]
we need the coc page before going online.

**Pull request merged**